easy_my_coop/models/coop.py

```diff
@@ -106,22 +117,13 @@ def check_empty_string(self, value):
             return False
         return True
 
-    def check_iban(self, iban):
-        validated = True
-        if iban:
-            try:
-                validate_iban(iban)
-            except ValidationError:
-                validated = False
-        return validated
-
     @api.multi
-    @api.depends('iban', 'skip_control_ng', 'is_company')
+    @api.depends('iban', 'skip_control_ng', 'is_company')  # why depend on is_company?

```

Comment:
@houssine78  why depend on is_company?

Reply:
legacy it was needed for national register number which was only only on personal subscription. So you was in need of knowing if you need to test it or not.

anyway it can be removed now